components/elm/src/main/elm_varcon.F90

```diff
@@ -249,7 +249,8 @@ subroutine elm_varcon_init()
     allocate( dzsoifl(1:nlevsoifl            ))
 
     if (use_extrasnowlayers) then
-       h2osno_max = 30000._r8
+        !h2osno_max = 1000._r8 
```

Comment:
can you please delete this commented out line?

Reply:
Done. Thanks for catching that.